config/webpack/webpack.config.js

```diff
@@ -108,17 +108,12 @@ const webpackConfig = (options, env, argv) => {
       }),
       new DuplicatePackageCheckerPlugin({
         showHelp: false,
+        alwaysEmitErrorsFor: ['react', 'react-dom', 'react-intl'],
```

Comment:
> react-intl

Oh, this might be an issue for any project not yet uplifted to terra-base v5 but is using terra-dev-site v4...

Reply:
Shouldn’t this also include disclosure manager and any of our other packages with context?

Reply:
Added the whole gang